layer.yaml

```diff
@@ -1,4 +1,5 @@
 includes: ['layer:basic']
+repo: git@github.com:juju-solutions/layer-docker.git
```

Comment:
Good point, Just converted that over to https. I didn't even stop to trip on the fact that will hang up non-collaborators. Whoops!